src/main.ts

```diff
@@ -588,6 +589,11 @@ export class QueryFeature extends SqlOpsFeature<undefined> {
 		};
 
 		let copyResults = (params: azdata.CopyResultsRequestParams): Thenable<azdata.CopyResultsRequestResult> => {
+			params.copyFromUIProcess = true;
```

Comment:
I don't think this is the right place to do this check, is there any reason you don't do this in STS?
Plus this code will be running in ADS, so I imagine `os.platform()` will return the OS reported by ADS processes, I believe we want to check the OS that STS is running from, right?

Reply:
So then I just keep the copyFromUIProcess flag, set that in the query editor from ADS. As for the changes in this file (src/main.ts), I get rid of all of this.